tests/e2e-utils/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@woocommerce/e2e-utils",
+    "version": "1.0.0",
+    "description": "End-To-End (E2E) test utils for WooCommerce",
+    "homepage": "https://github.com/woocommerce/woocommerce/tree/master/tests/e2e-utils/README.md",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/woocommerce/woocommerce.git"
+    },
+    "license": "GPL-3.0+",
+    "main": "index.js",
```

Comment:
This would need to be updated to 
```
"main": "build/index.js",
"module": "build-module/index.js",
```

Both the above folders need to be added to `.gitignore`